ndm-operator.yaml

```diff
@@ -666,49 +724,56 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: ndm-cluster-exporter-service
+  name: openebs-ndm-cluster-exporter-service
   namespace: openebs
   labels:
-    name: openebs-ndm-exporter
-    component: openebs-ndm-cluster-exporter
+    name: openebs-ndm-clsuter-exporter-service
```

Comment:
This was the common label used for monitoring @kmova which we decided to keep as `app: openebs-ndm-exporter`. The changes are there in https://github.com/openebs/node-disk-manager/pull/647 but not merged.